src/Pages/welcome.jsx

```diff
@@ -0,0 +1,12 @@
+import { useState } from "react";
+import Wave from "../Assets/koala-hug.gif"
+
+export default () => {
+  const [name, setname] = useState(localStorage.getItem("UserName") || "There");
```

Comment:
```suggestion
  const [name, setName] = useState(localStorage.getItem("UserName") || "There");
```
Another instance of naminb conventions

Reply:
Fixed e97eae75b859e2f1f70c09d6abe315d63c384f20